client/v2/helpers/home.go

```diff
@@ -21,12 +26,19 @@ func GetNodeHomeDirectory(name string) (string, error) {
 	}
 
 	// get the home directory from the environment variable
-	homeDir := os.Getenv("NODE_HOME")
+	// to not clash with the $HOME system variable, when no prefix is set
+	// we check the NODE_HOME environment variable
+	homeDir, envHome := "", "HOME"
+	if len(EnvPrefix) > 0 {
+		homeDir = os.Getenv(EnvPrefix + "_" + envHome)
+	} else {
+		homeDir = os.Getenv("NODE_" + envHome)
```

Comment:
This solve the bug that effectively if you want to set another home using $HOME, you always get the directory called `.simapp`.

```
HOME=~/download/.simapp simd config home 
/home/user/download/.simapp/.simapp
```

If we make the env check on home, earlier then we would have had

```
simd config home
/home/user
```

So the fix is using node home if you want to specify another folder than simapp when you have no prefix

```
NODE_HOME=~/download/simappv2 simd config home
/home/user/download/simappv2
```

and 

```
simd config home
/home/user/.simapp
```